README.md

```diff
@@ -43,6 +43,13 @@ Running spring app locally and db in docker
 1. Run `gradle bootRun --args='--spring.profiles.active=dev'`
 1. view site at http://localhost:8080
 
+Running spring app locally and db in docker on port 5433
+
+1. `cd backend`
+1. Run ` docker-compose --env-file .env.development up db`
+1. Run ` SR_DB_PORT=5433 gradle bootRun --args='--spring.profiles.active=dev'`
+1. view site at http://localhost:8080
```

Comment:
[ ** WIP NOTES *** ]

If Java isn't installed you can get it from `brew`: 
```sh
brew tap adoptopenjdk/openjdk
brew cask install adoptopenjdk8
brew install gradle
```

Run these commands:
```sh
cd backend
docker-compose --env-file .env.development up db
SR_DB_PORT=5433 gradle bootRun --args='--spring.profiles.active=dev'
```